### DISCUSSION TITLE
How to match group name for package rule?

### DISCUSSION THREAD

**Discussion by azban** [renovatebot/renovate#15770] [Request Help] [answered]
Hi,

I'm looking to apply additional configuration to a monorepo group that is coming out of `config:base`. Is there a way to match by the group name, rather than recreating the match rules that are used in the monorepo config?

Thanks

**Comment by astellingwerf** [comment, marked as answer]
You can create a package rule, that contains `"extends" : ["monorepo:acot"]`.
